BO4E/ENUM/WertBedeutungZusatz.cs

```diff
@@ -0,0 +1,50 @@
+using BO4E.meta;
+
+namespace BO4E.ENUM
+{
+    /// <summary>
+    /// Represents the Status Code (SG10 SGS 4405) 
+    /// </summary>
+    [NonOfficial(NonOfficialCategory.UNSPECIFIED)]
+    public enum WertBedeutungZusatz
+    {
+        /// <summary> T1 </summary>
+        TARIF_1,
+
+        /// <summary> T2 </summary>
+        TARIF_2,
+
+        /// <summary> T3 </summary>
+        TARIF_3,
+
+        /// <summary> T4 </summary>
+        TARIF_4,
+
+        /// <summary> T5 </summary>
+        TARIF_5,
+
+        /// <summary> T6 </summary>
+        TARIF_6,
+
+        /// <summary> T7 </summary>
+        TARIF_7,
+
+        /// <summary> T8 </summary>
+        TARIF_8,
+
+        /// <summary> T9 </summary>
+        TARIF_9,
```

Comment:
ist denn die anzahl der tarife fachlich begrenzt? Also ist sichergestellt, dass hier später nicht tarif 62,63,64 kommt?

Reply:
Ich denke, dass hier gar nichts sicher ist. Das sind die Möglichkeiten, die in der Version 2.3c genannt sind.